charts/kubearchive/templates/api_server/api_server.yaml

```diff
@@ -28,12 +28,13 @@ spec:
           image: {{ required "An API Server container image is required" .Values.apiServer.image }}
           resources:
             requests:
-              cpu: 200m
+              # Currently the API is sitting at 630m on performance tests
+              cpu: 700m
```

Comment:
Why change the request?  If I remember correctly, the request is just what is needed to start, but the limit it the max.  Do we always want to start with the limit?

Reply:
No real reason, I just changed both. I reverted it back to 200m because it makes sense, we won't be using 700m always, we are sitting at 630m under 100 requests per second, which I don't think we will hit often. Lets see.